Erstgespräch 2304.md

```diff
@@ -0,0 +1,43 @@
+# Erstgespräch 23.04. #
+
+## Aufgabenstellung ##
+
+**Entwicklung eines SW‐Systems zur Unterstützung der Planung von Präsentationen im Kontext des Moduls SE II.**
+
+Inhalt des Gesprächs war die Anforderungsanalyse mit dem Auftraggeber um die Rahmenbedingungen des Projektes zu erörtern.
+
+Zeit: 13.20 - 14.30 
+
+## Anwesenheit ##
+
+- Frau Hauptmann
+- Leo Lindhorst
+- Oliver von Seydlitz
+- Denis Keiling
+- Hung Nguyen
+- Fabian Krehnke
+
+
+Aktuell wird eine Excel-Tabelle für die Organisation genutzt. Diese liegt nur dem Auftraggeber vor. In dieser Tabelle sind die möglichen Zeiten für einen Termin festgehalten. Der Raum, in dem dieser stattfindet ist noch nicht bekannt. Termine und Termin Änderungen werden über E-Mail und Telefon in Absprache mit dem Auftraggeber ausgemacht und von ihm in die Liste eingetragen.
+
+**Problem**: der Prozess bis zum festen Termin ist zeitaufwendig, da viel kommuniziert werden muss und Zeiträume durch die Prüfungsphase beschränkt sind.

```

Comment:
Hier sollten die genauen Formulierungen hin, auf die wir uns geeinigt haben, siehe Mitschrift von @denisk176 . Evtl. auch was zum Vorgehen, wie wir zu den Formulierungen gekommen sind (Zettel usw.)

Reply:
Was ich hier meinte: Wir hatten uns in dem Treffen ja auf ganz konkrete Formulierungen für Problem und Ziel geeinigt. Da das erste Treffen ja noch @denisk176 mitgeschrieben hast und nicht du, frag ihn bitte was diese genauen Formulierungen waren